docs/config_file.md

```diff
@@ -221,6 +221,27 @@ the default one. Normal Docker behavior applies, so:
 - If only `image:latest` is specified, then Docker won't look in Docker Hub.
 - If the tag is omitted, then Docker will use the `latest` tag.
 
+If you specify a tag but no image name, `cross` will use the default image with
+the tag you provided:
+
+```toml
+[target.aarch64-unknown-linux-gnu]
+# Translates to `ghcr.io/cross-rs/aarch64-unknown-linux-gnu:edge`
+image = ":edge"
+
+[target.x86_64-unknown-linux-musl]
+# Translates to `ghcr.io/cross-rs/x86_64-unknown-linux-musl@sha256:77db671d8356a64ae72a3e1415e63f547f26d374fbe3c4762c1cd36c7eac7b99`
+image = "@sha256:77db671d8356a64ae72a3e1415e63f547f26d374fbe3c4762c1cd36c7eac7b99"
+```
+
+You can also specify a subtarget with no tag nor image name:
+
+```toml
+[target.x86_64-unknown-linux-gnu]
+# Translates to `ghcr.io/cross-rs/x86_64-unknown-linux-gnu:0.3.0-centos`
```

Comment:
I'm still not sure which tag should be shown as the default there. I didn't want to give users the idea that `main` is always the default, so I picked what should be the first tagged release with this feature.

Reply:
that's good, could add a release replacement [here](https://github.com/cross-rs/cross/blob/53a45dbf5819ebc86e0c71152757fcdad7e81d51/Cargo.toml#L77) to automatically update

Reply:
Oh neat, I didn't know about this! Will do.